### PR TITLE
chore: release react 0.6.0 & client 0.3.0

### PR DESCRIPTION
**Pull request by krzysu**

### CODE REVIEW

File: packages/gated-content/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lens-protocol/gated-content",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (cesarenaldi) [inline packages/gated-content/package.json:3]: @desfero what do you think we should do here? It seems ok to me to have this package live its own versioning until we reach v1.0.0, which will re-sync all of them.

File: packages/react-web/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lens-protocol/react-web",
-  "version": "0.4.1",
+  "version": "0.6.0",
```
Comment (cesarenaldi) [inline packages/react-web/package.json:3]: This one is kinda takes over from `@lens-protoco/react` for all users sung the Lens SDK on a React Web app. Seems ok to have it v0.6.0. Less opportunity for confusions.

Thoughts?

cc @desfero , @krzysu

**Comment by krzysu** [comment]
packages released, can I get approval to merge this to the `main`? @cesarenaldi @desfero

File: packages/gated-content/package.json (hunk 1 above)
Comment (cesarenaldi) [reply packages/gated-content/package.json]: it's a dependency of `react` and `react-web`.

**Review by cesarenaldi — APPROVED** [review]

**Pull request merged**